examples/web/index.html

```diff
@@ -1,147 +1,226 @@
-﻿<!doctype html>
+﻿<!DOCTYPE html>
 
 <html xmlns="http://www.w3.org/1999/xhtml">
-
-<head>
+  <head>
     <meta charset="utf-8" />
     <title>desktopJS</title>
     <script type="text/javascript" src="desktop.js"></script>
     <script type="text/javascript" src="desktopjs-openfin.js"></script>
     <script type="text/javascript" src="desktopjs-electron.js"></script>
 
     <!-- Bootstrap/JQuery from CDN to make example simple -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
-
-    <link rel="stylesheet" href="assets/css/app.css">
-</head>
-
-<body>
+    <link
```

Comment:
gah

Reply:
See the rational here: https://prettier.io/docs/en/options.html#print-width. While all of this is a change away from what was there before, I expect you would start to get used to it over time and it does enable the PRs to be much cleaner. e.g. if someone comes along and changes one of the properties below, just that line is changed. Similarly if someone adds a property. Particularly useful for side by side code changes. However, we can look at it changing. 

For what it's worth this isn't some arbitrary number that a someone has come up with and no one is using, other large groups are using it e.g. 
- https://github.com/facebook/react/blob/master/.prettierrc.js 
- https://github.com/webpack/webpack/blob/master/.prettierrc.js 

Though I expect pushback internally within MS if we make this a recommended style...

Reply:
Let's give it a try